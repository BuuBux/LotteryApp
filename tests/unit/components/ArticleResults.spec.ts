import ResultArticle from '@/components/ResultArticleComponent.vue';
import { mount } from '@vue/test-utils';
import testSetup from '../testSetup';

const propsData = { type: 'eurojackpot', selectedDate: 'Sep 04 2020' };
const createWrapper = () => mount(ResultArticle, {
  ...testSetup,
  propsData,
});

describe('Testing ResultArticleComponent.vue', () => {

  it('Create a snapshot', () => {
    expect(createWrapper()).toMatchSnapshot();
  });

  it('Check is selected date is string and not null', () => {
    const wrapper = createWrapper();
  });
});
