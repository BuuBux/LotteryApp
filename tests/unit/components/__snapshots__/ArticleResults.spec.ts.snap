// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ResultArticleComponent.vue Create a snapshot 1`] = `
<div class="c-numbers__wrapper">
  <div class="c-panel">
    <!---->
    <!---->
    <!---->
  </div>
</div>
`;
